packages/office-ui-fabric-react/src/components/DatePicker/__snapshots__/DatePicker.test.tsx.snap

```diff
@@ -25,7 +25,7 @@ exports[`DatePicker renders default DatePicker correctly 1`] = `
   <div
     aria-expanded={false}
     aria-haspopup="true"
-    aria-owns="DatePicker-Callout0"
+    aria-owns="DatePicker-Callout1"
```

Comment:
Is this change here expected?

Reply:
Yes, it's because a new ID is being generated now before this one for this test (see line 125 in this same file, the `DatePicker0-label` now being generated in the constructor.)